libs/subliminal_patch/providers/subssabbz.py

```diff
@@ -76,6 +78,14 @@ def get_matches(self, video):
              matches.add('hash')
 
         matches |= guess_matches(video, guessit(self.filename, {'type': self.type}))
+
+        if self.fps and video.fps and float(self.fps) != float(video.fps):
+            logger.info('FPS do not match for %r, %s vs %s. Reducing score.', self.page_link, self.fps, video.fps)
+            return set([x for x in matches if x in ['series', 'season', 'episode']])
```

Comment:
Sorry, but I don't agree with your statement that 50% of subtitles are 24fps. As stated above, 23.796 and 23.798 are not 24 fps and over the course of 1.5 hour movie, this gives a 5-8 seconds difference. That said, contrary to your statement, the majority of the subtitles out there are 2.976, not 24 fps.

Also I don't agree with the statement that the FPS parameter in a subtitle is usually the least reliable, quite the opposite - it is very reliable. Quite often there are multiple releases in a single archive that have different start/beginning offsets, not different framerates.

As for `['series', 'season', 'episode']`, if either of these are missing, the subtitle match will be completely rejected (https://github.com/morpheus65535/bazarr/blob/development/bazarr/get_subtitle.py#L339). And we don't want this - we want to biump down the subtitle rating, not reject it. With movies, this logic lowers the rating to as low as 0%, but the match is still shown, e.g. if the match is the only viable option for your video file FPS, you would still see it in matches.

Reply:
I would tend to accept this one. What is the downside of it? Anything other than having low scoring subtitles in manual search?

Reply:
How many standard movie frame rates do you think exists? By 24fps, I mean 24000/1001. I don't like these modification because they are trying to fix non existing problem, by a dirty hack which only break up the providers. This code will reduce the score of the tv shows, but will remove all movies with wrong frame rate. But the frame rate parameter is not essential any more as it is a relic from the times were most popular subtitle formats where frame based (microdvd). The funny thing is that these files can be easily converted to whatever frame rate do you want.  There is already pysubs2 include in bazarr. It's even more fun because this is already done. :) I think bazarr always save subtitles in srt format and in case of microdvd file it'll convert it, if the first line inside .sub file is it's frame rate. These are the reason why I think fps is not important requirement for scoring the results. I also think that the author of the PR is just experimenting with the code without really understanding the problem he is trying to solve.